src/popover.coffee

```diff
@@ -234,8 +234,15 @@ Ember.Widgets.PopoverComponent.reopenClass
   rootElement: '.ember-application'
   hideAll: -> $(document).trigger('popover:hide')
 
-  popup: (options) ->
-    @hideAll()
+  ###*
+   * Shows a popup with options used for PopoverComponent.
+   * @param {Object} options All options used for PopoverComponent.
+   * @param {boolean} [hideOthers=true] Indicates if other popover should be hidden when a new one
+   *    is shown. By default, it's set to true.
+  ###
+  popup: (options, hideOthers = true) ->
```

Comment:
I can see what Billy came from. It also makes sense. I am not sure which way is better though. 
About the `options.hideOthers` and `@hideOthers`, when we pass the `hideOthers` into `options` in the `popup` function, it should then set the `@hideOthers` to that value.